samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/common.c

```diff
@@ -0,0 +1,147 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <board.h>
+
+#include "common.h"
+#include "device_composition.h"
+#include "publisher.h"
+
+struct device *led_device[4];
+struct device *button_device[4];
+
+static struct k_work button_work;
+
+#define MINDIFF 2.25e-308
+
+float sqrt(float square)
```

Comment:
This should probably be added to Zephyr instead, others might want to use it! Not a big deal though